src/ui/profile_preferences.ui

```diff
@@ -42,7 +42,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>2</number>
```

Comment:
This needs to go back to `0` so that the first tab is shown initially - it is a side effect of not closing/saving  the form with that _tab_ being the selected/current one when Qt Designer (plugin in Qt Creator or the stand-alone application) is used to edit this form.